src/component/admin/billing/BillingGrid/BillingHistory/BillingHistory.tsx

```diff
@@ -0,0 +1,125 @@
+// TODO: finish this
+
+import {
+    Table,
+    SortableTableHeader,
+    TableBody,
+    TableCell,
+    TableRow,
+    TablePlaceholder,
+} from 'component/common/Table';
+import { PageContent } from 'component/common/PageContent/PageContent';
+import { DateCell } from 'component/common/Table/cells/DateCell/DateCell';
+import { useMemo, VFC } from 'react';
+import { useGlobalFilter, useSortBy, useTable } from 'react-table';
+import { sortTypes } from 'utils/sortTypes';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { IconButton, Typography } from '@mui/material';
+import FileDownload from '@mui/icons-material/FileDownload';
+
+interface IBillingHistoryProps {
+    data: Record<string, any>[];
+    isLoading?: boolean;
+}
+
+const columns = [
```

Comment:
This _might_ need to be memoized according to react-table docs: https://react-table.tanstack.com/docs/api/useTable

Reply:
Currently declared as an un-memoized top level const array, just like on the new Feature Toggle table, where I based this off of. 
What do you think @Tymek - Should we still memoize this?

Reply:
It should be fine if it is static, outside of the component.